source/isaaclab/isaaclab/sim/simulation_cfg.py

```diff
@@ -92,8 +92,14 @@ class PhysxCfg:
     """Enable a second broad-phase pass that makes it possible to prevent objects from tunneling through each other.
     Default is False."""
 
-    enable_stabilization: bool = True
-    """Enable/disable additional stabilization pass in solver. Default is True."""
+    enable_stabilization: bool = False
+    """Enable/disable additional stabilization pass in solver. Default is False.
+
+    .. note::
+
+        Setting this flag to True should only be done if the simulation steps are very large (less than 30Hz).
```

Comment:
Should we throw a warning about this finding from the code? This way users know that if they see some issues in forces, they have a heads up?